src/controllers/commands/CommandController.cpp

```diff
@@ -476,12 +476,33 @@ void CommandController::initialize(Settings &, Paths &paths)
     this->registerCommand("/usercard", [](const auto &words, auto channel) {
         if (words.size() < 2)
         {
-            channel->addMessage(makeSystemMessage("Usage /usercard [user]"));
+            channel->addMessage(
+                makeSystemMessage("Usage /usercard [user] (channel)"));
```

Comment:
I propose sticking to man page syntax (does this syntax have an official name?) for describing command parameters. In this case, that would mean:
```suggestion
                makeSystemMessage("Usage: /usercard user [channel]"));
```

Reply:
We already use a different syntax for optional parameters. Maybe we could switch, but I it's or of scope of this PR

Reply:
Yeah, I got the [required] (optional) from the /user command, but if <required> [optional] is the way going into the future this should probably be changed, no need to stick to the old one then.